ambiata-mafia.cabal

```diff
@@ -22,6 +22,7 @@ library
                     , attoparsec                      >= 0.12       && < 0.14
                     , base16-bytestring               == 0.1.*
                     , bytestring                      == 0.10.*
+                    , Cabal                           >= 1.22       && < 1.25
```

Comment:
I can feel the CPP clouds forming 

![](https://lh3.ggpht.com/oy_trWmjINZxAUZVA1gdDXeiyAtNNP_p-JOkOGJ2__deRhlOY68_Y0tF0An0f31R_Q=h900)